tests/rpc/calls/blockNumber.test.ts

```diff
@@ -0,0 +1,17 @@
+import { blockNumberHandler } from '../../../src/rpc/calls/blockNumber'
+import { RPCResponse } from '../../../src/types/types'
+
+describe('Test block number request testnet', () => {
+  it('Returns block number', async () => {
+    const request = {
+      jsonrpc: '2.0',
+      method: 'eth_blockNumber',
+      params: [],
+      id: 1,
+    }
+    const result: RPCResponse = <RPCResponse>await blockNumberHandler(request)
+
+    expect(typeof result.result).toBe('number')
```

Comment:
https://github.com/keep-starknet-strange/rosettanet/blob/main/rpc-call-examples/eth_blockNumber.hurl

Response should be in fex format and string